earthaccess/auth.py

```diff
@@ -235,6 +264,9 @@ def _get_credentials(
                 return False
             print("You're now authenticated with NASA Earthdata Login")
             self._credentials = (username, password)
+            self.username = username
+            self.password = password
```

Comment:
is this to make it easier for users to get/see the username+password (given they're stored via `self._credentials`)?

Reply:
I think `self._credentials` should probably go away, yes this way is to make it clearer.